code/AdobeEdge.brs

```diff
@@ -570,15 +570,10 @@ function _adb_task_node_EventProcessor(internalConstants as object, task as obje
                 return
             end if
 
-            ecid = m.stateManager.getECID()
-            configId = m.stateManager.getConfigId()
-            edgeDomain = m.stateManager.getEdgeDomain()
-
-
             requestId = event.uuid
             xdmData = event.data
 
-            m.edgeRequestWorker.queue(requestId, xdmData, event.timestamp_in_millis, configId, ecid, edgeDomain)
+            m.edgeRequestWorker.queue(requestId, xdmData, event.timestamp_in_millis)
```

Comment:
if everything we want is coming from event, why not just pass the event and extract requesstID, xdmData, ts inside the queue.

Reply:
If so, `RequestWorker` has to know the event object structure. It should be clearer to keep `Event` operations inside the event processor.

Reply:
sounds good.